_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,47 @@
+
+In practice, this means that to be able to interchange seamlessly between your Core and BU transaction history, you need 2 copies of the blockchain data; one for the Core chain and one for the BU chain. Same goes with the Armory database, as it is intimately tied to the underlying block data. However, you do not need to rescan the BU chain to get the copy of pre-fork Core database to work off of it. To be absolutely safe, you should use rebuild and rescan the databases for both chains after the fork, and even fully redownload the blockchain to ensure that everything is correct.
+
+To summarize, you need:
+
+- 1 copy of the blockchain to run exclusively against the Core client, with the associated Armory database directory.
+- 1 copy of the blockchain to run exclusively against the BU client, with associated Armory database directory.
+
+You can swap between the two by pointing Armory to the relevant pair of block data folder and database folder by using the `--satoshi-datadir` and `--dbdir` command line options respectively.
+
+### Avoiding Transaction Replay
+
+A transaction replay attack is where one transaction can end up confirmed in both blockchains thus resulting in coins on one chain being spent when you did not want those to be spent. This can happen either accidentally or intentionally where someone broadcasts your transaction to both networks.
+
+In order to avoid transaction replay, you must "taint" your coins; you want to get a transaction on one chain that is not valid on the other. Once you have that, you can mix all of your remaining coins with the "tainted" output to the same effect. At this point none of your UTXOs on one chain are valid on the other, which guarantees that your transactions are immune to transaction replay attacks.
+
+Tainting on BU is actually fairly easy. They want to increase the block size limit and reduce tx fees. At the same time, the departure of hash power from the main chain will result in slower Core blocks and thus less capacity and higher fees.
+
+Hence, the simplest way to taint on BU is this:
+
+- Create a transaction sending coins back to yourself, with a fee low enough such that it will quickly mine on BU and will probably not be mined quickly on Core.
+
+- Once the transaction is mined on BU, RBF the underlying output on the Core chain with a sufficiently high fee. This transaction can only be mined on Core now since it is invalid on BU (to which the transaction appears as a double spend).
+
+- Once your RBF transaction receives enough confirmations on the Core side -- six should be sufficient, you will have a UTXO exlusive to the BU chain and a UTXO exclusive the Core chain. You are now safe to taint the rest of your coins on each chain, keeping in mind that untainted coins remain valid on both chains. Including untainted coins in a transaction with tainted coins will be sufficient for tainting the untainted coins.
```

Comment:
Good point. I would point out that, in a nightmare scenario, it could take days to confirm transactions. Is this okay to add to the end of the paragraph?

"Please note that, in a worst case scenario, the miner hashrate drop could mean days pass before a block is added to a chain. While unlikely, this means weeks, or even months, could pass before coins can be tainted safely."